src/test/java/de/hpi/bpt/argos/api/EntityMappingEndpointTest.java

```diff
@@ -0,0 +1,355 @@
+
+		JsonObject newMapping = createNewMappingJson();
+		newMapping.addProperty("TargetStatus", targetStatus);
+
+		request.setContent(serializer.toJson(newMapping));
+
+		assertEquals(HttpStatusCodes.SUCCESS, request.getResponseCode());
+
+		List<EventEntityMapping> mappings = PersistenceAdapterImpl.getInstance().getEventEntityMappingsForEventType(testEventType.getId());
+		assertEquals(entityMappings + 1, mappings.size());
+
+		EventEntityMapping createdMapping = mappings.get(mappings.size() - 1);
+		assertMapping(createdMapping, targetStatus);
+
+		List<String> webSocketMessages = webSocket.awaitMessages(1, 1000);
+		ArgosTestUtil.assertWebSocketMessage(webSocketMessages.get(0), PersistenceArtifactUpdateType.CREATE, "EventEntityMapping");
+	}
+
+	@Test
+	public void testCreateEntityMapping_InvalidEntityTypeId_BadRequest() {
+		RestRequest request = RestRequestFactoryImpl.getInstance().createPostRequest(ARGOS_REST_HOST, getCreateEntityMappingUri());
+
+		JsonObject newMapping = createNewMappingJson();
+		newMapping.addProperty("EntityTypeId", 0);
+
+		request.setContent(serializer.toJson(newMapping));
+
+		assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());
+	}
+
+	@Test
+	public void testCreateEntityMapping_InvalidEventTypeId_BadRequest() {
+		RestRequest request = RestRequestFactoryImpl.getInstance().createPostRequest(ARGOS_REST_HOST, getCreateEntityMappingUri());
+
+		JsonObject newMapping = createNewMappingJson();
+		newMapping.addProperty("EventTypeId", 0);
+
+		request.setContent(serializer.toJson(newMapping));
+
+		assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());
+	}
+
+	@Test
+	public void testCreateEntityMapping_InvalidEntityAttributeTypeId_BadRequest() {
+		RestRequest request = RestRequestFactoryImpl.getInstance().createPostRequest(ARGOS_REST_HOST, getCreateEntityMappingUri());
+
+		JsonObject newMapping = createNewMappingJson();
+		newMapping.get("EventEntityMappingConditions")
+				.getAsJsonArray().get(0).getAsJsonObject()
+				.addProperty("EntityTypeAttributeId", 0);
+
+		request.setContent(serializer.toJson(newMapping));
+
+		assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());
+	}
+
+	@Test
+	public void testCreateEntityMapping_InvalidEventAttributeTypeId_BadRequest() {
+		RestRequest request = RestRequestFactoryImpl.getInstance().createPostRequest(ARGOS_REST_HOST, getCreateEntityMappingUri());
+
+		JsonObject newMapping = createNewMappingJson();
+		newMapping.get("EventEntityMappingConditions")
+				.getAsJsonArray().get(0).getAsJsonObject()
+				.addProperty("EventTypeAttributeId", 0);
+
+		request.setContent(serializer.toJson(newMapping));
+
+		assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());
+	}
+
+	@Test
+	public void testDeleteEntityMapping() throws Exception {
+		EventEntityMapping testMapping = createMapping();
+		int entityMappings = getMappingsCount();
```

Comment:
How about "entityMappingsCount" as a name for each of those variables? I got pretty confused when reading.